ui/hooks/useDisplayName.ts

```diff
@@ -15,6 +16,7 @@ export type UseDisplayNameResponse = {
   name: string | null;
   hasPetname: boolean;
   contractDisplayName?: string;
+  image?: string | null;
```

Comment:
I am not super happy to use image here since hook name is also `useDisplayName`. But I think this perfect place to implement this because image is going to be used in the same component. 
Maybe we can think to rename it to `useDisplayNameProps`